Makefile

```
@@ -1,4 +1,4 @@
-export COMPOSE_PROJECT_NAME=yii-queue
+export COMPOSE_PROJECT_NAME=queue
```

```suggestion
export COMPOSE_PROJECT_NAME=yii-queue
```

This name is used in globlal namespace. I think, better keep `yii-` prefix.